crates/rethnet_evm/src/state/debug.rs

```diff
@@ -1,9 +1,38 @@
+use std::{fmt::Debug, ops::Deref};
+
 use auto_impl::auto_impl;
 use rethnet_eth::{Address, B256, U256};
 use revm::primitives::{AccountInfo, Bytecode};
 
-/// Function type for modifying account information.
-pub type AccountModifierFn = Box<dyn Fn(&mut U256, &mut u64, &mut Option<Bytecode>) + Send>;
+/// Debuggable function type for modifying account information.
+pub struct AccountModifierFn {
+    inner: Box<dyn Fn(&mut U256, &mut u64, &mut Option<Bytecode>) + Send>,
+}
+
+impl AccountModifierFn {
+    /// Constructs an [`AccountModifierDebuggableFn`] from the provided function.
+    pub fn new(func: Box<dyn Fn(&mut U256, &mut u64, &mut Option<Bytecode>) + Send>) -> Self {
+        Self { inner: func }
+    }
+}
+
+impl Debug for AccountModifierFn {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(
+            f,
+            "{}",
+            std::any::type_name::<dyn Fn(&mut U256, &mut u64, &mut Option<Bytecode>)>()
```

Comment:
I'm curious why this is useful instead of just formatting it as, say, `rethnet_evm::state::debug::AccountModifierFn` or something.

Reply:
I just wanted to see the internal function handle: parameter and return types